src/components/ArticleFooter.js

```diff
@@ -2,14 +2,13 @@ import { h } from 'preact'
 import { useI18n } from 'hooks'
 import { buildWpMobileWebUrl, canonicalizeTitle } from 'utils'
 
-export const ArticleFooter = ({ lang, title, items = [] }) => {
+export const ArticleFooter = ({ lang, title, items = [], headerTitle }) => {
```

Comment:
the `lang` here already present as `contentLang`, so we can do the similar things in `useArticle` to have the interface language in `i18n.locale`, then use `setlocale( lang )` to get the header title here.

Reply:
Not a bad idea, this simplifies `Article.js` then